demo/src/screens/incubatorScreens/WheelPickerScreen.tsx

```diff
@@ -109,16 +91,24 @@ export default () => {
           selectedValue={selectedMonth}
         />
 
-        <Text h3>Years</Text>
-        <View width={'100%'}>
+        <View row bottom marginT-s5>
+          <Text h3>Years</Text>
+          <Button bodySmall marginL-s2 link label="Reset" size={Button.sizes.xSmall} onPress={resetInitialYear}/>
```

Comment:
Yes that's because we're resetting to the same value twice. 
I wasn't sure if to keep this reset button or remove it. 
I needed something to check the functionality. 
But I don't want to overcomplicated the example with this.. 
WDYT?

Reply:
If you want to keep it simple then I think we should remove it.

Reply:
Done